### PR TITLE
fix debug_traceTransaction bugs

### PR DESCRIPTION
**Pull request by vitaliy-io**
https://github.com/iotaledger/isc-private/issues/214

Issue 1 was fixed as part of https://github.com/iotaledger/wasp/pull/3502

### CODE REVIEW

File: packages/evm/jsonrpc/tracer_call.go [hunk 1]
```diff
@@ -35,8 +36,8 @@ type callLog struct {
 type CallFrame struct {
 	Type         vm.OpCode       `json:"-"`
 	From         common.Address  `json:"from"`
-	Gas          uint64          `json:"gas"`
-	GasUsed      uint64          `json:"gasUsed"`
+	Gas          string          `json:"gas"`
+	GasUsed      string          `json:"gasUsed"`
```
Comment (dessaya) [inline packages/evm/jsonrpc/tracer_call.go]: Maybe you want `hexutil.Uint64` here?

File: packages/evm/jsonrpc/tracer_call.go [hunk 2]
```diff
@@ -46,7 +47,7 @@ type CallFrame struct {
 	Logs         []callLog       `json:"logs,omitempty" rlp:"optional"`
 	// Placed at end on purpose. The RLP will be decoded to 0 instead of
 	// nil if there are non-empty elements after in the struct.
-	Value            *big.Int `json:"value,omitempty" rlp:"optional"`
+	Value            string `json:"value,omitempty" rlp:"optional"`
```
Comment (dessaya) [inline packages/evm/jsonrpc/tracer_call.go]: and `hexutil.Big`

**Review by dessaya — APPROVED** [review]

File: packages/evm/jsonrpc/tracer_call.go (hunk 1 above)
Comment (vitaliy-io) [reply packages/evm/jsonrpc/tracer_call.go]: Thanks, did not notice these types

File: packages/evm/jsonrpc/tracer_call.go (hunk 2 above)
Comment (vitaliy-io) [reply packages/evm/jsonrpc/tracer_call.go]: fixed

File: packages/evm/jsonrpc/tracer_call.go (hunk 1 above)
Comment (vitaliy-io) [reply packages/evm/jsonrpc/tracer_call.go]: fixed

**Pull request merged**